src/components/VersionChooser/index.spec.tsx

```diff
@@ -27,24 +73,54 @@ describe(__filename, () => {
   });
 
   it('splits the list of versions into listed and unlisted lists', () => {
-    const listedVersions = [
+    const addonId = 999;
+    const listedVersions: ExternalVersionsList = [
       {
-        ...fakeVersions[0],
+        ...fakeVersionsList[0],
         channel: 'listed',
       },
     ];
-    const unlistedVersions = [
+    const unlistedVersions: ExternalVersionsList = [
       {
-        ...fakeVersions[1],
+        ...fakeVersionsList[1],
         channel: 'unlisted',
       },
     ];
 
-    const root = render({ versions: [...listedVersions, ...unlistedVersions] });
+    const store = configureStore();
+    _loadVersionsList(store, addonId, [...listedVersions, ...unlistedVersions]);
+
+    const root = render({ addonId, store });
 
     root.find(VersionSelect).forEach((versionSelect) => {
       expect(versionSelect).toHaveProp('listedVersions', listedVersions);
       expect(versionSelect).toHaveProp('unlistedVersions', unlistedVersions);
     });
   });
+
+  it('dispatches fetchVersionsList() on mount', () => {
+    const addonId = 888;
+    const store = configureStore();
+    const dispatch = spyOn(store, 'dispatch');
+    const fakeThunk = createFakeThunk();
+    const _fetchVersionsList = fakeThunk.createThunk;
+
+    render({ _fetchVersionsList, addonId, store });
+
+    expect(dispatch).toHaveBeenCalledWith(fakeThunk.thunk);
+    expect(_fetchVersionsList).toHaveBeenCalledWith({ addonId });
+  });
+
+  it('does not dispatch fetchVersionsList() on mount when lists are already loaded', () => {
+    const addonId = 888;
+    const store = configureStore();
+    _loadVersionsList(store, addonId, []);
```

Comment:
It could be useful to add another test that loads versions for the wrong addonId to check that versions are re-fetched.